components/butterfly/protocols/swim.proto

```diff
@@ -51,6 +51,7 @@ message Service {
   optional string pkg = 9;
   optional bytes cfg = 10;
   optional SysInfo sys = 12;
+  optional string channel = 13;
```

Comment:
I don't think we need to include the channel the service is leveraging in this rumor. This would spread the information to other Supervisors but that's not a requirement for the EventSrv protocol

Reply:
Yeah I'm gonna scrap this PR and send the channel/service information in a different message since the channel is not included in census data and not intended to be gossiped as such